Create academics-grants-round page

samajammin opened the pull request:
<!--- Provide a general summary of your changes in the Title above -->

## Description

Draft grants page

Preview deploy:
https://ethereumorgwebsitedev01-academicsgrantsround.gtsb.io/en/community/grants/academics-grants-round

**Note: we need this page to be live first:** 
https://esp.ethereum.foundation/academics-grants

## Related Issue

None
<!--- This project accepts pull requests related to open issues -->
<!--- If suggesting a new feature or change, please discuss it in an issue first -->
<!--- If fixing a bug, there should be an issue describing it with steps to reproduce -->
<!--- Please link to the issue here: -->

corwintines reviewed (commented):
Just wondering if this should be linked out to somewhere on the site, or if it is just a page we are having that can be sent out to people.

wackerow commented:
Do we know when the https://esp.ethereum.foundation/academics-grants page will be live? Still getting 404 on this page... Would suggest we leave this in draft to prevent merging before that link is live.

samajammin commented:
> Just wondering if this should be linked out to somewhere on the site, or if it is just a page we are having that can be sent out to people.

Good call, added a link from the /community/grants/ page 👍 

> Do we know when the https://esp.ethereum.foundation/academics-grants page will be live? Still getting 404 on this page... Would suggest we leave this in draft to prevent merging before that link is live.

Should be live by tomorrow but agreed, just updated this to a draft PR 👍

minimalsm commented on src/content/community/grants/academics-grants-round/index.md:
```diff
@@ -0,0 +1,147 @@
+</ExpandableCard>
+
+<ExpandableCard
+contentPreview="Up to $750.000 in total"
```
```suggestion
contentPreview="Up to $750,000 in total"
```

minimalsm commented on src/content/community/grants/academics-grants-round/index.md:
```diff
@@ -0,0 +1,147 @@
+contentPreview="Up to $750.000 in total"
+title="What is the budget available for this round?">
+
+   <p>This grants round has up to $750.000 in total, which is to be distributed among selected projects. </p>
```
```suggestion
   <p>This grants round has up to $750,000 in total, which is to be distributed among selected projects. </p>
```

minimalsm commented on src/content/community/grants/academics-grants-round/index.md:
```diff
@@ -0,0 +1,147 @@
+  This round is now open - apply before April 22, 2022! <a href="https://esp.ethereum.foundation/academics-grants/" target="_blank">Apply here</a>.
+</InfoBanner>
+
+_The Ethereum Foundation is sponsoring a wave of grants to support academic work related to Ethereum. This grants round has up to $750.000 in total available. Proposals are due <b>April 22, 2022</b>. Here are all the details you need._
```
```suggestion
_The Ethereum Foundation is sponsoring a wave of grants to support academic work related to Ethereum. This grants round has up to $750,000 in total available. Proposals are due <b>April 22, 2022</b>. Here are all the details you need._
```

minimalsm reviewed (commented):
This should use $750,000 instead of $750.000 if we're denominating USD
